src/envs/pybullet_env.py

```diff
@@ -329,37 +329,81 @@ def _create_move_end_effector_to_pose_option(
         or closed according to the finger status.
         """
 
-        def _policy(state: State, memory: Dict, objects: Sequence[Object],
-                    params: Array) -> Action:
-            del memory  # unused
-            # First handle the main arm joints.
+        def _initiable(state: State, memory: Dict, objects: Sequence[Object],
+                       params: Array) -> bool:
+            # This is a really hacky hack to handle the situation where we are
+            # planning with a simplified model (that from the Blocks env) where
+            # states do not have joints. In this case, we want the option to
+            # be always initiable. Furthermore, we use pragma here because
+            # the oracle approach is not unit tested on this environment
+            # because it would be too slow.
+            if not hasattr(state, "joint_state"):  # pragma: no cover
+                return True
+            # Extract the current joint state.
+            current_joints = cast(_PyBulletState, state).joint_state
+            # Make a plan in joint space.
             current, target, finger_status = \
                 get_current_and_target_pose_and_finger_status(
                     state, objects, params)
-            # Run IK to determine the target joint positions.
+            target_joints = self._pybullet_robot.run_inverse_kinematics(
+                target, validate=True)
```

Comment:
why do we want validate=True here? remember the comment you deleted: "Furthermore, updating the state of the robot during simulation, which validate=True would do, is discouraged by PyBullet." so this seems like a bad thing to do?